web/package.json

```diff
@@ -16,8 +16,7 @@
     "@emotion/react": "^11.8.2",
     "@emotion/styled": "^11.8.1",
     "@mui/icons-material": "^5.5.1",
-    "@mui/lab": "^5.0.0-alpha.75",
-    "@mui/material": "^5.5.3",
+    "@mui/material": "5.12.2",
```

Comment:
i locked this version bc 5.13.3 breaks a test i think due to a datepicker. could not figure out why.